spec/controllers/v2/concerns/sortable_spec.rb

```diff
@@ -22,14 +22,19 @@
 # `:result` field is an array of indexes into entities array with expected order, nested arrays will be sorted by id
 #  e.g.: expected order is 0, 1, 2, 3, but 2 and 3 need to be sorted by FactoryBot ID (sorted property is same for both)
 #     => written as [0, 1, [2, 3]]
+#
+# parents - array of parental ActiveRecord models
+# extra_params - extra parameters to be passed into request
```

Comment:
This is not descriptive enough, where and how these would go? One of them is a `let` the other one is passed as an argument of the shared example. Also it is not explained what they actually do...